src/main/solr/dssolr/conf/solrconfig.xml

```diff
@@ -820,12 +819,13 @@
       <str name="spellcheck.collateExtendedResults">true</str> > <!-- Probably disable later.But usefull to see what is in data -->
       <str name="spellcheck.maxCollationTries">10</str>
       <str name="spellcheck.maxCollations">5</str>
-      <str name="spellcheck.build">true</str>              
+      <str name="spellcheck.build">false</str>  <!-- Much overhead building it every query. Not recommended and will not scale -->              
```

Comment:
Just a question, when and how does the spellcheck get build? Is it build and rebuild during indexing or do we need to manually build it somehow?